mule-module-apikit/src/main/java/org/mule/module/apikit/validation/RestXmlSchemaValidator.java

```diff
@@ -188,13 +188,15 @@ private static void setFeatures(DocumentBuilderFactory dbf)
             feature = "http://xml.org/sax/features/external-parameter-entities";
             dbf.setFeature(feature, externalEntities);
 
+            feature = "http://apache.org/xml/features/nonvalidating/load-external-dtd";
+            dbf.setFeature(feature, externalEntities);
+
             feature = "http://apache.org/xml/features/disallow-doctype-decl";
             dbf.setFeature(feature, !expandEntities);
```

Comment:
Since _"http://apache.org/xml/features/disallow-doctype-decl"_ is disabled _"http://apache.org/xml/features/nonvalidating/load-external-dtd"_ shouldn't be needed we have a test for this at **org.mule.module.apikit.schema.XxeAttackTestCase**